src/common/GPOWrapper/GPOWrapper.h

```diff
@@ -62,6 +62,7 @@ namespace winrt::PowerToys::GPOWrapper::implementation
         static winrt::hstring GPOWrapper::GetConfiguredMwbPolicyDefinedIpMappingRules();
```

Comment:
Offtopic: I had tested tested it. But it looks strange that this is referencing `GPOWrapper` class here. 🤔